phoenix-core/src/main/java/org/apache/phoenix/mapreduce/MultiHfileOutputFormat.java

```diff
@@ -265,7 +265,8 @@ private WriterLength getNewWriter(final String tableName , byte[] family, Config
                                         .withBytesPerCheckSum(CompatUtil.getBytesPerChecksum(conf))
                                         .withBlockSize(blockSize)
                                         .withDataBlockEncoding(encoding)
-                                        .withCellComparator(CellComparatorImpl.COMPARATOR);
+                                        .withCellComparator(CellComparatorImpl.COMPARATOR)
+                                        .withCreateTime(System.currentTimeMillis());;
```

Comment:
This should probably be EnvironmentEdgeManager.currentTime() so that tests can override the timestamp correctly.